GMX-Bots/Large-Open-and-Increase-Positions/src/network.ts

```diff
@@ -0,0 +1,38 @@
+import { VAULT_CONSTANTS } from "./constants";
+const { VAULT_ARBITRUM, VAULT_AVALANCHE } = VAULT_CONSTANTS;
+
+interface NetworkData {
+  vaultAddress: string;
+}
```

Comment:
It would be nice if you can add a different threshold per each network

Reply:
Added `threshold` in NetworkData.